docs/remote/devcontainer-cli.md

```diff
@@ -1,220 +1,140 @@
-* `--image-name` : The Remote - Containers extension typically determines its own name for the images it builds. You can specify the name to use for the built image using the `--image-name` option.
+You can then run commands in this dev container:
 
-You can also type `devcontainer build --help` to see a full list of available options.
+```bash
+devcontainer exec --workspace-folder <path-to-vscode-remote-try-rust> cargo run
+```
 
-### **Optional:** Avoiding problems with images built using Docker
+This will compile and run the Rust sample, outputting:
 
-Given Dockerfiles and Docker Compose files can be used without VS Code or the VS Code `devcontainer` CLI, you may want to let users know that they should not try to build the image directly if it will not work as expected. To solve this problem, you can add a build argument that needs to be specified for things to work.
+```bash
+[33 ms] dev-containers-cli 0.1.0.
+   Compiling hello_remote_world v0.1.0 (/workspaces/vscode-remote-try-rust)
+    Finished dev [unoptimized + debuginfo] target(s) in 1.06s
+     Running `target/debug/hello_remote_world`
+Hello, VS Code Remote - Containers!
+{"outcome":"success"}
+```
 
-For example, you could add the following to your Dockerfile:
+These steps above are also provided in the CLI repo's [README](https://github.com/devcontainers/cli/blob/main/README.md).
 
-```Dockerfile
-ARG vscode
-RUN if [[ -z "$vscode" ]] ; then printf "\nERROR: This Dockerfile needs to be built with VS Code !" && exit 1; else printf "VS Code is detected: $vscode"; fi
-```
+## Automation
 
-And the following in your `devcontainer.json`:
+If you'd like to use the devcontainer CLI in your CI/CD builds or test automation, you can find examples of GitHub Actions and Azure DevOps Tasks in the [devcontainers/ci](https://github.com/devcontainers/ci) repository.
 
-```json
- "build": {
-      "dockerfile": "Dockerfile",
-      "args": {
-          // set vscode arg for Dockerfile
-          "vscode": "true"
-      },
-    }
-```
+## Feedback
 
-In the Docker Compose case, you can add this argument to a separate [override file to extend your configuration](/docs/remote/create-dev-container.md#extend-your-docker-compose-file-for-development) that is located in a different place in your source tree than the primary Docker Compose file.
+The devcontainer CLI and specification are under active development and we welcome your feedback, which you can provide in [this issue](https://github.com/devcontainers/cli/issues/7), or through new issues and pull requests in the [devcontainers/cli](https://github.com/devcontainers/cli) repository.
 
 ## Next steps
 
+* [Dev container specification repository](https://github.com/devcontainers/spec) - File and review issues to shape the direction of development containers and the dev container CLI.
```

Comment:
```suggestion
* [Dev container specification repository](https://github.com/devcontainers/spec) - File and review issues to shape the direction of development containers and the dev container CLI by reviewing and contributing to the open specification.
```

Reply:
I like to have Next steps lists avoid line breaks so how about: "Read and contribute to the open specification"
There has been plenty of mention of what the specification covers earlier